client/my-sites/menus/style.scss

```diff
@@ -738,6 +738,44 @@
 				font-size: 14px;
 				padding: 10px 13px;
 
+				&.menu-item-options__term-selector {
+					padding: 10px 0;
+					overflow-y: hidden;	
+
+					.term-selector {
+						border: 0;
+					}
+
+					.term-selector__results {
+						&::-webkit-scrollbar {
```

Comment:
I'm mimicking the scrollbar logic seen below to try to make the scroll bar look like it does in other menu areas.  I'm fine with removing it.